nodewatcher/modules/analysis/channel_allocation/channel_lookup.py

```diff
@@ -0,0 +1,63 @@
+from nodewatcher.core.monitor import models
+
+
+# TODO: Turn hard-coded arrays into a node-specific (stemming from regulatory practices) property.
+ch_2ghz = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]
+ch_5ghz = [36, 38, 40, 42, 44, 46, 48, 50, 52, 54, 56, 58, 60, 62, 64, 100, 102, 104, 106, 108, 110, 112, 114, 116,
+           118, 120, 122, 124, 126, 128, 132, 134, 136, 138, 140, 142, 144, 149, 151, 153, 155, 157, 159, 161, 165]
+highest_2ghz_channel = max(ch_2ghz)
```

Comment:
Can you make this then be used in the other file as well? Instead of having a constant there?